ddl-scripts/create_tables_yugabyte.sql

```diff
@@ -20,7 +20,8 @@ CREATE TABLE IF NOT EXISTS event_journal(
   PRIMARY KEY((slice, entity_type) HASH, persistence_id, seq_nr ASC)
 );
 
-CREATE INDEX IF NOT EXISTS event_journal_slice_idx ON event_journal(entity_type ASC, slice ASC, db_timestamp ASC);
+CREATE INDEX IF NOT EXISTS event_journal_slice_idx ON event_journal(entity_type ASC, slice ASC, db_timestamp ASC)
+  INCLUDE (seq_nr, deleted);
```

Comment:
So `seq_nr` and `deleted` goes in the index but without being keys of the index or what does `INCLUDE` do?

Reply:
Yes, good that you spotted that, I should have mentioned it. That will be much cheaper because then the backtracking queries don't have to do the join (lookup) with the parent table (possibly on a remote tablet). It will only use the index.

Reply:
Is this an specific Yugabyte thing? 
That's quite interesting actually.

Reply:
yup: https://docs.yugabyte.com/latest/api/ysql/the-sql-language/statements/ddl_create_index/#include-clause

It's a way to create a consistent materialized view via an index.

One could include all columns in the index, at the cost of larger size on disk (and maybe slightly slower writes). Nice thing is that a user of this plugin could change the index definition if that is preferred and all queries would still be the same.